src/rp2_common/pico_flash/flash.c

```diff
@@ -9,6 +9,7 @@
 #if PICO_FLASH_SAFE_EXECUTE_PICO_SUPPORT_MULTICORE_LOCKOUT
 #include "pico/multicore.h"
 #endif
+#include "pico/time.h"
```

Comment:
Looking at the location of the lines you quoted in your warning message, I guess this include ought to be move inside the `#if PICO_FLASH_SAFE_EXECUTE_USE_FREERTOS_SMP` section?

Reply:
I'm fine with it as i I think